runtime/src/benchmark.rs

```diff
@@ -15,25 +15,25 @@
 
 frame_benchmarking::define_benchmarks!(
     [frame_system, SystemBench::<Runtime>]
-    [pallet_assets, Assets]
-    [pallet_balances, Balances]
-    [pallet_session, SessionBench::<Runtime>]
+    [cumulus_pallet_parachain_system, ParachainSystem]
     [pallet_timestamp, Timestamp]
-    [pallet_message_queue, MessageQueue]
-    [pallet_sudo, Sudo]
-    [cumulus_pallet_xcmp_queue, XcmpQueue]
-    [pallet_scheduler, Scheduler]
-    [pallet_preimage, Preimage]
     [pallet_proxy, Proxy]
-    [cumulus_pallet_parachain_system, ParachainSystem]
-    [pallet_multisig, Multisig]
     [pallet_utility, Utility]
-    [pallet_treasury, Treasury]
-    [pallet_evm, EVM]
+    [pallet_multisig, Multisig]
+
+    [pallet_balances, Balances]
+    [pallet_assets, Assets]
+
+    [pallet_sudo, Sudo]
+
+    [pallet_colleator_selection, CollatorSelection]
```

Comment:
```suggestion
    [pallet_collator_selection, CollatorSelection]
```

Reply:
Ok, it's a left over.... I fixed the spelling in the other places but I forget this one